ZipUtilitiesTests/NOZSwiftTests.swift

```diff
@@ -157,4 +157,15 @@ func TearDownZipQueue()
             // NSLog("Progress: \(progress)%")
         })
     }
+    
+    func testCompletionClosureIsCalled() {
```

Comment:
so there is a race condition in this test.  The issue is that an operation will "finish" before it calls "complete".  That means there can be a race between the operation finishing and you getting your expectation fulfilled.  You can see this in the unit test failure with Travis CI.

2 options:

1) just extend the timeout for your waitForExpectationsWithTimeout.  Have it be big enough like 10 seconds.
2) remove the waitUntilFinished and just use the waitForExpectationsWithTimeout with a large enough timeout like 10 seconds.


Thanks for the PR and I look forward to merging it, once the unit test is ready!